artifactory/commands/utils/configfile.go

```diff
@@ -375,41 +415,47 @@ func (configFile *ConfigFile) setUseNugetV2() {
 	configFile.Resolver.NugetV2 = coreutils.AskYesNo("Use NuGet V2 Protocol?", false)
 }
 
-// Check correctness of spec file configuration
```

Comment:
Check correctness of spec file configuration
-->
Validate spec file configuration